dockerfiles/roxie/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG BUILD_VER
+FROM hpccsystems/platform-core:${BUILD_VER}
+
+USER hpcc
+RUN mkdir -p /var/lib/HPCCSystems/roxie
+WORKDIR /var/lib/HPCCSystems/roxie
+RUN roxie --init > roxie.json
```

Comment:
Possibly ok for a demo, but I don't think you'll want to do this long term.

Reply:
Depends whether we think that we want to supply the "default" config in text form for inspection/documentation or not.

Reply:
I thought we said the embedded config would have the defaults, and the shipped config would have them all commented out - so they wouldn't match.

Reply:
Yes, but I'm __think__ that the output from --init  should quite possibly be the latter rather than the former...